### ISSUE TITLE
Nightly Builds

### ISSUE THREAD

**Issue by JesterOrNot** [gosh-terminal/gosh#162]
Add nightly builds automated with CI/CD

**Comment by JesterOrNot** [comment]
Nightly builds did not trigger last night.

**Comment by JesterOrNot** [comment]
@RDIL when you have a chance can you take a look at the nightly builds

**Comment by RDIL** [comment]
[RDILBot Automated Response] Successfully added "can you take a look at the nightly builds" to Reece’s to-do list!

**Comment by RDIL** [comment]
oh apparently this is on my todo list so ill take a look now

**Comment by RDIL** [comment]
Looks to be working now.

**Comment by JesterOrNot** [comment]
It's not Not logged in to https://index.docker.io/v1/
check the latest build @RDIL and the package registry

**Comment by RDIL** [comment]
Looking at the logs, it says `Not logged in to index.docker.io/v1`

**Comment by RDIL** [comment]
Yeah, if you want to also publish to docker hub you need an account and to set up a second action that publishes to it, because right now your action is only set up to upload to the GPR.
